Lesson1/Dariia-Skoryk/task1_vector/test_vector/tst_testcase.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <gtest/gtest.h>
+#include <gmock/gmock-matchers.h>
+#include "../source/main.h"
+
+using namespace testing;
+
+TEST(testCase, testSet)
+{
+    std::vector<int> firstVector, secondVector, resultVector;
+    EXPECT_TRUE(resultVector == combineVectors(firstVector.begin(), firstVector.end(),
+                                               secondVector.begin(), secondVector.end()));
+}
+
+TEST(testCase_2, testSet)
+{
+    std::vector<int> firstVector, secondVector, resultVector;
```

Comment:
It is acceptable in tests, but don't declare several objects in one line. Separate it.